internal/nodes/avi_model_nodes.go

```diff
@@ -704,6 +704,9 @@ func (v *AviVsNode) CalculateCheckSum() {
 	if v.EnableRhi != nil {
 		checksum += utils.Hash(utils.Stringify(*v.EnableRhi))
 	}
+	for _, vsvipref := range v.VSVIPRefs {
```

Comment:
@aaha97 could you please rebase with the current master. It now has changes related to graph checksum computation, so including this in the Vs Checksum calculation should not be required anymore.
cc: @sudswasavi

Reply:
done